src/services/monaco-editor.service.ts

```diff
@@ -81,6 +81,15 @@ export class MonacoEditorService {
 			});
 	}
 
+  /**
+   * Dispose all editor models
+   */
+  disposeModels() {
```

Comment:
Should we name it `reset` or something? To be able to do other cleanup stuff as well if needed?

Reply:
Not sure what else needs clean up. I rather thought there might be a corresponding `disposeModel(uri)` method in the future. Then it would probably be good to have `disposeModel()`

Your call.

Reply:
Yes, you are right.